app/models/need.rb

```diff
@@ -57,21 +57,25 @@ def start_required_for_assessment
   scope :completed, -> { where(status: :complete) }
   scope :uncompleted, -> { where.not(status: [:complete, :cancelled]) }
   scope :started, lambda { |start_on|
-    date = Date.today
-    if start_on.present?
-      if start_on == 'tomorrow'
-        date = Date.tomorrow
-      elsif start_on == 'next_week'
-        date += 7.days
-      elsif start_on == 'any_time'
-        date = nil
-      end
+    next all if start_on.nil? || start_on == 'any_time'
+    case start_on
+    when 'today'
+      where('start_on >= ? and start_on <= ?', Date.today, Date.today.end_of_day)
+    when 'tomorrow'
+      where('start_on >= ? and start_on <= ?', Date.today + 1.days, Date.today.end_of_day + 1.days)
+    when 'next_week'
+      where('start_on >= ? and start_on <= ?', Date.today + 1.days, Date.today.end_of_day + 7.days)
```

Comment:
I think you can just write `Date.tomorrow` but it does not matter

Reply:
Ah good spot, yeah that would be better